src/PrestaShopBundle/Controller/Admin/Sell/CustomerService/MerchandiseReturnController.php

```diff
@@ -119,15 +128,36 @@ public function editAction(int $orderReturnId, Request $request): Response
 
             return $this->redirectToRoute('admin_merchandise_returns_index');
         }
+        $allowPrintingOrderReturnPdf =
+            $editableOrderReturn->getOrderReturnStateId() === OrderReturnSettings::ORDER_RETURN_STATE_WAITING_FOR_PACKAGE_ID;
 
-        return $this->render('@PrestaShop/Admin/Sell/CustomerService/OrderReturn/edit.html.twig', [
+        return $this->render('@PrestaShop/Admin/Sell/CustomerService/MerchandiseReturn/edit.html.twig', [
+            'allowPrintingOrderReturnPdf' => $allowPrintingOrderReturnPdf,
+            'editableOrderReturn' => $editableOrderReturn,
```

Comment:
Similarly here, all the data you need should be accessible via the `form.vars.value` in twig no need to perform to call and have this variable injected in the template (especially if you use it only to get `editableOrderReturn.orderReturnId` you already have this ID in `$orderReturnId`